standalone/runtime/src/lib.rs

```diff
@@ -20,24 +20,23 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 // `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
-#![recursion_limit = "256"]
-
+#![recursion_limit = "512"]
```

Comment:
Why change this？

Reply:
We added many pallets to stakepoolv2, and when compiling, it accrued an error and suggest us to widen the limit to 512.